homeassistant/runner.py

```diff
@@ -121,9 +121,7 @@ def run(runtime_config: RuntimeConfig) -> int:
         try:
             _cancel_all_tasks_with_timeout(loop, TASK_CANCELATION_TIMEOUT)
             loop.run_until_complete(loop.shutdown_asyncgens())
-            # Once cpython 3.8 is no longer supported we can use the
-            # the built-in loop.shutdown_default_executor
-            loop.run_until_complete(_shutdown_default_executor(loop))
+            loop.shutdown_default_executor()
```

Comment:
This is a coroutine function.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.shutdown_default_executor

```suggestion
            loop.run_until_complete(loop.shutdown_default_executor())
```